Sources/PerformanceTesting/PerformanceTestCase.swift

```diff
@@ -26,7 +26,8 @@ open class PerformanceTestCase: XCTestCase {
 
     // MARK: Instance Methods
 
-    /// Benchmarks the performance of a closure.
+    /// - Returns: An array of two-tuples containing the size of the structure and the average time
```

Comment:
per slack discussion, could you maybe just change the wording from `size of the structure` to `input size`? I'm planning a major refactor and the stuff you've added in this PR will be helpful.